utils/user_management.py

```diff
@@ -0,0 +1,158 @@
+'''
+Utility for managing user names and passwords for TabPy.
+'''
+
+from argparse import ArgumentParser
+import logging
+import os
+import random
+import sys
+from tabpy_server.app.util import parse_pwd_file
+from tabpy_server.handlers.util import hash_password
+
+logger = logging.getLogger(__name__)
+
+
+def build_cli_parser():
+    parser = ArgumentParser(
+        description=__doc__,
+        epilog='''
+            For more information about how to configure and
+            use authentication for TabPy read documentation
+            at https://github.com/tableau/TabPy
+            ''',
+        argument_default=None,
+        add_help=True)
+    parser.add_argument(
+        'command',
+        choices=['add', 'update'],
+        help='Command to execute')
+    parser.add_argument(
+        '-u',
+        '--username',
+        help='Username to add to passwords file')
+    parser.add_argument(
+        '-f',
+        '--pwdfile',
+        help='Passwords file')

```

Comment:
Should be "Fully qualified path to passwords file"

Reply:
fixed